src/views/self/view_weapons.cpp

```diff
@@ -17,6 +17,7 @@ namespace big
 		ImGui::BeginGroup();
 
 		components::command_checkbox<"infammo">();
+		components::command_checkbox<"alwaysfullammo">();
```

Comment:
might want to emphasize difference between infinite ammo

Reply:
What? This is just the internal command name. The user will see "Always Full Ammo" and the description of "Refills your ammo every tick." What would you have me change it to?

Reply:
Nah it's fine as-is.